keyboards/handwired/neodox/rules.mk

```
@@ -0,0 +1,20 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System controliouy
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+WPM_ENABLE = yes
+SPLIT_KEYBOARD = yes
+QUANTUM_PAINTER_ENABLE = yes
+SERIAL_DRIVER = vendor
+WS2812_DRIVER = vendor
+QUANTUM_PAINTER_DRIVERS = gc9a01_spi
+DEBOUNCE_TYPE = asym_eager_defer_pk
```

Enabled feature should be handled in info.json
```suggestion
BACKLIGHT_DRIVER = pwm
SERIAL_DRIVER = vendor
WS2812_DRIVER = vendor
QUANTUM_PAINTER_DRIVERS = gc9a01_spi
DEBOUNCE_TYPE = asym_eager_defer_pk
```